assets/src/edit-story/elements/gif/icon.js

```diff
@@ -14,19 +14,25 @@
  * limitations under the License.
  */
 
-/**
- * External dependencies
- */
-import { __ } from '@web-stories-wp/i18n';
-
 /**
  * Internal dependencies
  */
-import { Media } from '../../icons';
+import StoryPropTypes from '../../types';
+import VisibleImage from '../media/visibleImage';
+import { getSmallestUrlForWidth } from '../media/util';
 
-function GifIcon() {
-  const alt = __('GIF element', 'web-stories');
-  return <Media aria-label={alt} />;
+function GifLayerIcon({
```

Comment:
For all of these I did the following:
- Renamed the `layer.js` to `icon.js` since the `LayerContent` is now the icon in the new designs
- Renamed `icon.js` to `layer.js`
- Edited the new `layer.js` to have the necessary text content